pkg/proxy/ipvs/proxier.go

```diff
@@ -2007,7 +2008,12 @@ func (proxier *Proxier) syncService(svcName string, vs *utilipvs.VirtualServer,
 
 func (proxier *Proxier) syncEndpoint(svcPortName proxy.ServicePortName, onlyNodeLocalEndpoints bool, vs *utilipvs.VirtualServer) error {
 	appliedVirtualServer, err := proxier.ipvs.GetVirtualServer(vs)
-	if err != nil || appliedVirtualServer == nil {
+	if err != nil {
+		klog.Errorf("Failed to get IPVS service, error: %v", err)
+		return err
+	}
+	if appliedVirtualServer == nil {
+		err = errors.New("IPVS virtual service does not exist")
```

Comment:
just:
```
return errors.New("IPVS virtual service does not exist")
```

Reply:
log lines can also be removed I think

Reply:
The second if block's log line has been removed, don't remove the log line in the first if block because I think it is necessary to info which ipvs handle failed.